### PR TITLE
Added more examples to Firebase Analytics Swift SDK readme

### PR DESCRIPTION
**Pull request by Skizzy-create**
Created custom event logging API and manual screen view event logging API significantly reduces the amount of code required for event logging:

Before: We had separate event logging codes for both the button click and screen view events, resulting in redundant lines of code.
After: With the custom event logging API and manual screen view event logging API, we have reduced the event logging code to a few lines, improving code efficiency and readability.
By using these APIs, you can easily log events and screen views throughout the app with concise and reusable code, making it more efficient and maintainable.

### CODE REVIEW

File: FirebaseAnalyticsSwift/README.md [hunk 1]
```diff
@@ -43,7 +43,77 @@ struct ContentView: View {
...
+        .analyticsScreen(name: "main_content")
+    }
+}
+
+// Introducing a custom event logging API.
+extension Analytics {
+    static func logCustomEvent(name: String, parameters: [String: Any]? = nil) {
+        Analytics.logEvent(name, parameters: parameters)
+    }
+}
```
Comment (morganchen12) [inline FirebaseAnalyticsSwift/README.md:92]: Does this wrapper provide any additional utility over the method it's wrapping?

**Review by morganchen12 — APPROVED** [review]
Looks good, had one small nit. Thanks!

File: FirebaseAnalyticsSwift/README.md (hunk 1 above)
Comment (paulb777) [reply FirebaseAnalyticsSwift/README.md:92]: We appreciate the suggestion, but this repo should focus on documenting the Firebase APIs as they are defined. 

The APIs have already been through a review process and it's fine for any particular app to wrap as they would like.

Please remove the `logCustomEvent` wrapper.

Comment (Skizzy-create) [reply FirebaseAnalyticsSwift/README.md:92]: ok
right on it

Comment (Skizzy-create) [reply FirebaseAnalyticsSwift/README.md:92]: I am removing the whole custom event logging API

Comment (Skizzy-create) [inline FirebaseAnalyticsSwift/README.md:92]: Going to remove this whole part

Comment (Skizzy-create) [reply FirebaseAnalyticsSwift/README.md]: take a look now, if i have to make any changes

Comment (Skizzy-create) [reply FirebaseAnalyticsSwift/README.md]: @paulb777 sir can we connect on Linkedin? Stange place to ask but i don't know how else to reach you

**Comment by Skizzy-create** [comment]
@paulb777 sir can we connect on Linkedin? Stange place to ask but i don't know how else to reach you

**Comment by Skizzy-create** [comment]
also if there are any other changes let me know

**Review by paulb777 — APPROVED** [review]
Looks good. Thanks @Skizzy-create 

You can reach me by email at paulbeusterien at google.com

**Pull request merged**